### PR TITLE
Upgrade core to 8.8

### PR DESCRIPTION
**Pull request by andypost**
- composer.json updated
- initial scaffold converted
- TODO: review https://www.drupal.org/project/drupal/issues/3064990

https://www.drupal.org/docs/develop/using-composer/using-drupals-composer-scaffold

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,11 +32,11 @@
     "ckeditor/panelbutton": "^4.10",
     "composer/installers": "^1.2",
     "cweagans/composer-patches": "^1.6.5",
-    "drupal-composer/drupal-scaffold": "^2.5",
     "drupal/better_normalizers": "^1.0@beta",
     "drupal/components": "^1",
     "drupal/config_installer": "^1",
-    "drupal/core": "^8.7.5",
+    "drupal/core-composer-scaffold": "^8.8.0",
+    "drupal/core-recommended": "^8.8.0",
```
Comment (andypost) [inline composer.json]: Not sure this change needed, looks better to stick to core

File: composer.json [hunk 2]
```diff
@@ -46,15 +46,15 @@
...
     "drupal/styleguide": "^1",
     "drupal/swiftmailer": "^1",
     "drupal/telephone_validation": "^2.2",
     "drupal/token": "^1",
     "drupal/views_bulk_edit": "^2.3",
     "drupal/webform": "^5.4",
-    "drush/drush": "^10.0.2",
+    "drush/drush": "^10.1.0",
     "webflo/drupal-finder": "^1.1",
     "webmozart/path-util": "^2.3",
```
Comment (andypost) [inline composer.json]: This one could be removed

**Comment by andypost** [comment]
One more example https://github.com/drupalcommerce/demo-project/commit/3feb6701140a694a192956f679fcfbb9c05b5185

File: composer.json [hunk 3]
```diff
@@ -32,11 +32,11 @@
     "ckeditor/panelbutton": "^4.10",
     "composer/installers": "^1.2",
     "cweagans/composer-patches": "^1.6.5",
-    "drupal-composer/drupal-scaffold": "^2.5",
     "drupal/better_normalizers": "^1.0@beta",
     "drupal/components": "^1",
     "drupal/config_installer": "^1",
-    "drupal/core": "^8.7.5",
+    "drupal/core-composer-scaffold": "^8.8.0",
```
Comment (davidferlay) [inline composer.json]: Reverting ` "drupal/core-composer-scaffold": "^8.8.0"` to `"drupal-composer/drupal-scaffold": "^2.5",` as it breaks : 
- `make info` 
- `composer install`

File: composer.json (hunk 1 above)
Comment (davidferlay) [reply composer.json]: Working like a charm atm, `make tests` passing successfully

File: composer.json (hunk 2 above)
Comment (davidferlay) [reply composer.json]: Removed

**Pull request merged**